env/HERA.env

```diff
@@ -76,6 +76,16 @@ elif [[ "${step}" = "aeroanlrun" ]]; then
     [[ ${NTHREADS_AEROANL} -gt ${nth_max} ]] && export NTHREADS_AEROANL=${nth_max}
     export APRUN_AEROANL="${launcher} -n ${npe_aeroanlrun}"
 
+elif [[ "${step}" = "landanlrun" ]]; then
+
+    export APRUNCFP="${launcher} -n \$ncmd ${mpmd_opt}"
```

Comment:
I know @RussTreadon-NOAA removed these lines from the `atmanlrun`. Please check if the land jobs will execute via MPMD as well?

Reply:
Removed these lines, and confirmed that there were not any effects for the `landanlrun`.